css/theme.css

```diff
@@ -42,6 +42,11 @@
 
 /* Let's use flexbox everywhere */
 
+body {
+  height: 100vh;
+  overflow: hidden;
+}
```

Comment:
Nit: I have no idea why that's necessary, maybe you could add some comment explaining.